connector/src/test/java/com/google/cloud/hive/bigquery/connector/IntegrationTests.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.hive.bigquery.connector;
+
+import static com.google.cloud.hive.bigquery.connector.TestUtils.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.hive.bigquery.connector.config.RunConf;
+import com.google.cloud.hive.bigquery.connector.config.RunConf.Config;
+import com.google.cloud.storage.*;
+import com.klarna.hiverunner.*;
+import com.klarna.hiverunner.builder.HiveShellBuilder;
+import com.klarna.hiverunner.config.HiveRunnerConfig;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.apache.commons.io.FileUtils;
+import org.apache.hadoop.hive.conf.HiveConf.ConfVars;
+import org.junit.jupiter.api.Test;
+import repackaged.by.hivebqconnector.com.google.common.collect.ImmutableList;
+
+// TODO: When running the tests, some noisy exceptions are displayed in the output:
+//  "javax.jdo.JDOFatalUserException: Persistence Manager has been closed".
+//  Those exceptions don't impact the execution of the tests, although they perhaps
+//  make them run a bit slower overall. This seems related to:
+//  https://issues.apache.org/jira/browse/HIVE-25261, which was fixed in Hive 4.0.0,
+//  so we might have to find a workaround to make those go away with Hive 3.X.X.
+
+public class IntegrationTests {
```

Comment:
Instead of calling this class `IntegrationTests`, can we be more specific?

Reply:
Yes good point. I actually already started that process in a later PR. So I'm going to close this comment here and will address that at a later point if that's ok.